POEApi.Model/POEApi.Model.csproj

```diff
@@ -1,126 +1,137 @@
-    <Compile Include="GemCategory.cs" />
-    <Compile Include="ImageComparer.cs" />
-    <Compile Include="Item.cs" />
-    <Compile Include="ItemFactory.cs" />
-    <Compile Include="JSONProxy\Stash.cs" />
-    <Compile Include="Map.cs" />
-    <Compile Include="OrbType.cs" />
-    <Compile Include="Events\POEEventArgs.cs" />
-    <Compile Include="Events\POEEventState.cs" />
-    <Compile Include="POEModel.cs" />
-    <Compile Include="Property.cs" />
-    <Compile Include="ProxyMapper.cs" />
-    <Compile Include="Requirement.cs" />
-    <Compile Include="Settings.cs" />
-    <Compile Include="ShopSetting.cs" />
-    <Compile Include="Socket.cs" />
-    <Compile Include="StackInfo.cs" />
-    <Compile Include="Stash.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Events\StashLoadedEventArgs.cs" />
-    <Compile Include="Tab.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\POEApi.Infrastructure\POEApi.Infrastructure.csproj">
-      <Project>{2F0E4301-694B-4A26-80D8-D57042DA9D6F}</Project>
-      <Name>POEApi.Infrastructure</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\POEApi.Transport\POEApi.Transport.csproj">
-      <Project>{5933C062-45EA-4BC3-9AE9-0D6CFC26D505}</Project>
-      <Name>POEApi.Transport</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup>
-    <Content Include="Buyouts.xml">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Data.xml">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-      <SubType>Designer</SubType>
-    </Content>
-    <Content Include="Settings.xml">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-      <SubType>Designer</SubType>
-    </Content>
-  </ItemGroup>
-  <ItemGroup>
-    <Content Include="Settings.xsd">
-      <SubType>Designer</SubType>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-  </ItemGroup>
-  <ItemGroup>
-    <Content Include="Data.xsd">
-      <SubType>Designer</SubType>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProductVersion>8.0.30703</ProductVersion>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>{9841DD07-224D-4534-B9D1-F934C1D7BC73}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>POEApi.Model</RootNamespace>
+    <AssemblyName>POEApi.Model</AssemblyName>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>

```

Comment:
There isn't a prereq for moving to 4.6.1 I just want to use the new language features.